src/lib/visualization/lqp_visualizer.cpp

```diff
@@ -89,37 +89,37 @@ void LQPVisualizer::_build_subtree(const std::shared_ptr<AbstractLQPNode>& node,
   }
 }
 
-void LQPVisualizer::_build_dataflow(const std::shared_ptr<AbstractLQPNode>& from,
-                                    const std::shared_ptr<AbstractLQPNode>& to, const InputSide side) {
+void LQPVisualizer::_build_dataflow(const std::shared_ptr<AbstractLQPNode>& from_node,
+                                    const std::shared_ptr<AbstractLQPNode>& to_node, const InputSide side) {
```

Comment:
How about `source_node` and `target_node`?

Reply:
Much better.